### PR TITLE
Add section on wallets / clients

### PR DESCRIPTION
**Pull request by OR13**
Purpose of this PR is to provide informative context related to the "software system used by holders"... We have a number of core data model concepts that are tied to "holder" that are awkward to discuss without some ability to speak to the relationship between "wallets and holders".

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/137.html" title="Last updated on Aug 16, 2023, 4:43 PM UTC (37d5f74)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/137/972fd68...37d5f74.html" title="Last updated on Aug 16, 2023, 4:43 PM UTC (37d5f74)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -340,6 +340,34 @@ <h2>Securing JSON-LD VCs with COSE</h2>
...
   </section>
 
+<section class="informative">
+  <h2>Wallets</h2>
+  <p>
+  <a data-cite="VC-DATA-MODEL#dfn-issuers">issuers</a>,
+  <a data-cite="VC-DATA-MODEL#dfn-holders">holders</a> and 
+  <a data-cite="VC-DATA-MODEL#dfn-verifier">verifiers</a> might rely on clients, 
+  as defined in <a href="https://datatracker.ietf.org/doc/html/rfc4949">RFC4949</a>, 
+  such clients are often referred to as wallets or digital credential wallets,
```
Comment (TallTed) [inline index.html:350]:
```suggestion
  as defined in <a href="https://datatracker.ietf.org/doc/html/rfc4949">RFC4949</a>.
  Such clients are often referred to as <i>wallets</i> or <i>digital credential wallets</i>,
```

File: index.html [hunk 2]
```diff
@@ -340,6 +340,34 @@ <h2>Securing JSON-LD VCs with COSE</h2>
...
+  <a data-cite="VC-DATA-MODEL#dfn-holders">holders</a> and 
+  <a data-cite="VC-DATA-MODEL#dfn-verifier">verifiers</a> might rely on clients, 
+  as defined in <a href="https://datatracker.ietf.org/doc/html/rfc4949">RFC4949</a>, 
+  such clients are often referred to as wallets or digital credential wallets, 
+  when they support storing and presenting digital credentials.
+  </p>
+  <p>
+  In order to meet <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> requirements, some
+  <a data-cite="VC-DATA-MODEL#dfn-issuers">issuers</a> might be required to assess the quality of a wallet used by a 
+  <a data-cite="VC-DATA-MODEL#dfn-holders">holder</a> prior to issuing and delivery credentials to a <a data-cite="VC-DATA-MODEL#dfn-holders">holder</a>.
```
Comment (TallTed) [inline index.html:356]:
```suggestion
  <a data-cite="VC-DATA-MODEL#dfn-issuers">issuers</a> might need to assess the quality of a wallet used by a 
  <a data-cite="VC-DATA-MODEL#dfn-holders">holder</a>, prior to issuing and
  delivering credentials to a <a data-cite="VC-DATA-MODEL#dfn-holders">holder</a>.
```

File: index.html [hunk 3]
```diff
@@ -340,6 +340,34 @@ <h2>Securing JSON-LD VCs with COSE</h2>
...
+  </p>
+  <p>
+  For example, some <a data-cite="VC-DATA-MODEL#dfn-verifier">verifiers</a> might require that cryptographic material 
+  associated with a <a data-cite="VC-DATA-MODEL#dfn-holders">holder</a>, be protected at specific 
+  assurance levels, 
+  see <a href="https://pages.nist.gov/800-63-3-Implementation-Resources/63B/AAL/">NIST 800-63-3: Authenticator Assurance Levels</a>.
+  </p>
+  <p class="note">
+    See <a href="https://datatracker.ietf.org/doc/draft-looker-oauth-attestation-based-client-auth/">
```
Comment (TallTed) [inline index.html:365]:
```suggestion
    Also see <a href="https://datatracker.ietf.org/doc/draft-looker-oauth-attestation-based-client-auth/">
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by OR13** [comment]
@TallTed thanks for the review!

**Review by TallTed — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Pull request merged**